charts/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 2.31.1
+version: 2.31.2
```

Comment:
```suggestion
version: 2.32.0
```

Reply:
last time it was done was from `2.30.14` to `2.30.15`, but it should be a minor version change?

Reply:
Yes it should be a minor version, I think last time was an oversight 😅